jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -1104,3 +1054,36 @@ def markStageUnstableIfPluginsFailedToBuild() {
         unstable('Some plugins failed to build. See the ./build.sh step for logs and more details')
     }
 }
+
+def triggerIntegrationTests(String buildManifestUrl, String buildManifestUrlOpenSearch) {
+    Boolean skipIntegTests = (INTEG_TEST_JOB_NAME == '' || TEST_MANIFEST == '' || buildManifestUrl == '')
```

Comment:
Please add a null check for `buildManifestUrlOpenSearch` as well.
Thanks!

Reply:
Thanks for noticing that,, adding it now.